ml/gguf.ksy

```diff
@@ -0,0 +1,274 @@
+meta:
+  id: gguf
+  title: GGML model file
+  file-extension: gguf
+  license: CC0-1.0
+  ks-version: 0.10
+  endian: le
+  bit-endian: le
+doc: |
+  GGUF is a file format for storing machine learning models for inference with
+  the GGML library, or executors based on GGML. Models are typically developed
+  using PyTorch or some other framework, and then converted to GGUF for use in
+  GGML.
+
+  GGUF is a successor file format to GGML, GGMF and GGJT, and is designed 
+  to be unambiguous by containing all the information needed to load a 
+  model. It is also designed to be extensible, so that new information 
+  can be added to models without breaking compatibility.
+doc-ref:
+  - https://github.com/ggerganov/ggml/blob/master/docs/gguf.md
```

Comment:
I guess this should be changed to https://github.com/ggml-org/ggml/blob/master/docs/gguf.md ?